en/micro-integrator/mkdocs.yml

```diff
@@ -297,6 +297,14 @@ nav:
             - 'Sample 1: Hello World': 'setup/deployment/k8s-samples/hello-world.md'
             - 'Sample 2: Message Routing': 'setup/deployment/k8s-samples/content-based-routing.md'
             - 'Sample 3: JMS Sender/Receiver': 'setup/deployment/k8s-samples/jms-sender-receiver.md'
+        - K8s Pipeline:
```

Comment:
Why is this not under Deploy on Kubernetes section? Is this a completely different use case? If so, how?